rllib/core/rl_module/rl_module.py

```diff
@@ -285,7 +285,19 @@ class RLModule(abc.ABC):
 
     def __init__(self, config: RLModuleConfig):
         self.config = config
+        # Make sure, `setup()` is only called once, no matter what. In some cases
```

Comment:
I noticed this happening iff certain conditions are fulfilled (multiple inheritance, wrong inheritance order). This is to make 100% sure the user's custom classes will NOT create 2 RLModules under the hood w/o noticing.